s3-source-connector/src/main/java/io/aiven/kafka/connect/s3/source/AivenKafkaConnectS3SourceConnector.java

```diff
@@ -59,8 +60,9 @@ public Class<? extends Task> taskClass() {
     @Override
     public List<Map<String, String>> taskConfigs(final int maxTasks) {
         final var taskProps = new ArrayList<Map<String, String>>();
-        for (int i = 0; i < maxTasks; i++) {
-            final var props = Map.copyOf(configProperties);
+        for (int i = 1; i <= maxTasks; i++) {
```

Comment:
Is there any particular reason why we need to start from 1 instead of 0?

Reply:
No, for some convenience I changed it, but looking at partition ids, which start at 0, I think it's better to start this at 0 too. I shall update.

Reply:
Updated task id start back to 0